server/bin/pbench-restore-and-unpack-tarballs.py

```diff
@@ -107,7 +107,7 @@ def main(options):
     if not archive_p.is_dir():
         print(
             f"The configured ARCHIVE directory, {archive}, is not a valid directory",
-            file=sys.stderr
+            file=sys.stderr,
```

Comment:
Dang, `black` really loves those trailing list commas. It's a bit obsessive.

Reply:
IIRC from the Black documentation, the trailing comma can be _added by the developer_ to prevent Black from un-wrapping the lines; so, once Black decides that it wants the lines wrapped, I guess it adds it to ensure a consistent and deterministic result (which, I suppose, makes good sense, sadly).